config/di.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+declare(strict_types=1);
+
+return [
```

Comment:
Does it make sense to provide the config if it is empty?

Reply:
Hi, have deleted it.